### PR TITLE
Fix #173, add tc39-ntoes folder for old link redirection

### LINKED ISSUE #173: Add tc39-notes directory with redirects
See https://github.com/tc39/Reflector/issues/251.

You could essentially copy all `*.html` files from https://github.com/tc39/notes/tree/gh-pages and add them to a new `tc39-notes` folder on the site hosted here. This would take care of redirecting old notes URLs to the new canonical URLs.

### PR DESCRIPTION
**Pull request by othree**
Fix #173. Tested on local env. The redirection works as expected.

### CODE REVIEW

**Comment by ljharb** [comment]
(if the OP of the post, not the title, has `fix #173`, then things will link properly)

File: tc39-notes/2012-05_may-21.html [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<!doctype html>
+<link rel="canonical" href="https://github.com/tc39/notes/blob/master/meetings/2012-05/may-21.md">
+<meta http-equiv=refresh content="0;URL=https://github.com/tc39/notes/blob/master/meetings/2012-05/may-21.md">
```
Comment (ljharb) [inline tc39-notes/2012-05_may-21.html]: can you add a trailing newline to all these files?

**Review by ljharb — APPROVED** [review]

File: tc39-notes/2012-05_may-21.html (hunk 1 above)
Comment (othree) [reply tc39-notes/2012-05_may-21.html]: newline added

**Comment by mathiasbynens** [comment]
Let's get this merged and deployed asap please :)

**Review by chicoxyzzy — APPROVED** [review]
👍

**Pull request merged**